functions/getWidgetContent.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+const getGitHubWidgetContent = require('./lib/get-github-widget-content');
+
+const widgetHandlerMap = {
+	github: getGitHubWidgetContent
+};
+
+const getWidgetContent = async ({ context, req }) => {
+  const {
+    query: { widget }
+  } = req;
+
+  if (!widgetHandlerMap[widget]) {
+	return {
+        status: "error",
+        code: 400,
+        error: "Widget type unrecognized or missing."
+      };
+  }
+
+  const widgetHandler = widgetHandlerMap[widget];
+  const payload = await widgetHandler({ context });
+
+  return {
+	status: "ok",
```

Comment:
I'm experimenting with a response wrapper here. This is the only endpoint using this response structure, for now.

Looking back at this, I don't like that `status` depends on the consumer having awareness of the available enums. All I want is a simple boolean value I can check to know if the query was OK. I will update this to use `ok`.